server/nSiteNVar_server.R

```diff
@@ -890,6 +939,52 @@ output$GRAPH_MAIN3c = renderDygraph({
     return(dg)
 })
 
+output$GRAPH_QC3c <- renderPlot({
+    
+    sites <- na.omit(isolate(input$SITES3[1:3]))
+    varA <- isolate(input$VARS3[1])
+    dmns <- isolate(get_domains3())
+    conc_unit <- isolate(input$CONC_UNIT3)
+    show_pchem <- isolate(input$SHOW_PCHEM3)
+    agg <- isolate(input$AGG3)
+    dates <- isolate(input$DATES3)
+    
+    # sites <<- na.omit(isolate(input$SITES3[1:3]))
+    # varA <<- isolate(input$VARS3[1])
+    # dmns <<- isolate(get_domains3())
+    # conc_unit <<- isolate(input$CONC_UNIT3)
+    # show_pchem <<- isolate(input$SHOW_PCHEM3)
+    # agg <<- isolate(input$AGG3)
+    # dates <<- isolate(input$DATES3)
+    
+    # reactive_vals$facet3aQC
+    reactive_vals$facet3a
+    
+    # streamdata <<- dataChem()
+    streamdata = dataChem()
+    
+    # dischargedata <<- dataQ()
+    dischargedata = dataQ()
+    
+    alldata <- inner_join(streamdata,
+                          dischargedata,
+                          by = c("datetime", "site_name")) %>%
+        rename(value=5) %>%
+        select(datetime, site_name, value, discharge)
+    
+    qc <- ggplot(alldata,
+                 aes(x = discharge, y = value, colour = site_name),
+                 environment=environment()) +
+        geom_point() +
+        scale_colour_manual(values = linecolors,
+                            breaks = c(sites)) +
+        labs(y = "") +
+        ggthemes::theme_few() +
```

Comment:
i would appreciate a rundown of ggplot (and ggthemes, etc) some time. been resisting learning that stuff for far too long